telco-core/networking/multinetworkpolicy/mnpEnable.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  useMultiNetworkPolicy: true
```

Comment:
Rather than have two separate views of the network.operator CR, can you add this setting to telco-core/networking/Network.yaml and add a comment with it to indicate that when enabled they should include the MultiNetworkPolicy CRs as well. That way we have a single reference config with available options. 
Thanks!